eng/doc/Downloads.md

```diff
@@ -0,0 +1,29 @@
+ linux-arm64 | - [Binaries (tar.gz)](https://aka.ms/golang/release/latest/go1.18.linux-arm64.tar.gz)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.18.linux-arm64.tar.gz.sha256)<br/>- [Signature<sup>1</sup>](https://aka.ms/golang/release/latest/go1.18.linux-arm64.tar.gz.sig)<br/> |- [Binaries (tar.gz)](https://aka.ms/golang/release/latest/go1.18-fips.linux-arm64.tar.gz)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.18-fips.linux-arm64.tar.gz.sha256)<br/>- [Signature<sup>1</sup>](https://aka.ms/golang/release/latest/go1.18-fips.linux-arm64.tar.gz.sig)<br/> | - [Binaries (tar.gz)](https://aka.ms/golang/release/latest/go1.17.linux-arm64.tar.gz)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.17.linux-arm64.tar.gz.sha256)<br/>- [Signature<sup>1</sup>](https://aka.ms/golang/release/latest/go1.17.linux-arm64.tar.gz.sig)<br/> |- [Binaries (tar.gz)](https://aka.ms/golang/release/latest/go1.17-fips.linux-arm64.tar.gz)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.17-fips.linux-arm64.tar.gz.sha256)<br/>- [Signature<sup>1</sup>](https://aka.ms/golang/release/latest/go1.17-fips.linux-arm64.tar.gz.sig)<br/> |
+ windows-amd64 | - [Binaries (zip)](https://aka.ms/golang/release/latest/go1.18.windows-amd64.zip)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.18.windows-amd64.zip.sha256)<br/> |- [Binaries (zip)](https://aka.ms/golang/release/latest/go1.18-fips.windows-amd64.zip)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.18-fips.windows-amd64.zip.sha256)<br/> | - [Binaries (zip)](https://aka.ms/golang/release/latest/go1.17.windows-amd64.zip)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.17.windows-amd64.zip.sha256)<br/> |- [Binaries (zip)](https://aka.ms/golang/release/latest/go1.17-fips.windows-amd64.zip)<br/>- [Checksum (SHA256)](https://aka.ms/golang/release/latest/go1.17-fips.windows-amd64.zip.sha256)<br/> |
+
+
+<!-- END TABLES -->
+
+> <sup>1</sup> - Validate GPG signatures using our public key:
+> * https://download.microsoft.com/download/f/a/2/fa2420dd-3f08-4621-82cf-5a22abcbc8f9/microsoft-managed-lang-compiler.asc
+> * Shortened link for sharing: https://aka.ms/managed-lang-compiler-public-key
+
+
+The links in this table "float": each one downloads the latest Go release built by Microsoft that matches that column's major release number. For example, a 1.17 link will download 1.17.8-1 if that is the latest version of 1.17 available.
+
+To download a specific version, create a static link by copying one of the above URLs and adding a patch and revision number to the end of the existing version number section. More aka.ms links are generated than the ones that appear in the table, because the table isn't updated after every single build. Browse recent GitHub releases in the microsoft/go repository to see the valid patch and revision numbers to use.
```

Comment:
Nit: could add an example here, such as how to change the url for 1.17 to 1.17.8-1